1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -19,12 +19,18 @@ instead!
 3. Call the function once, giving it the object `cartForParty` as an argument.
    Use `console.log` to display the result.
 -----------------------------------------------------------------------------*/
-const cartForParty = {
-  // TODO complete this function
+const cartForParty = {chips:0.99 ,appel:1 ,orange:2,rice:5,mac:6
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
-  // TODO replace this comment with your code
+function calculateTotalPrice(cartObject) {
+  let amount=0;
+const values=Object.values(cartObject);
+  for(let i=0;i< values.length;i++){
+amount += values[i];
+  }
+ const message = `Total: €${amount}`;
+ return message;
+ 
```

Comment:
good job! another way to this exercise could be following a more functional programming approach:

```suggestion
function calculateTotalPrice(object) {
  const amount = Object.values(object).reduce((totalPrice, price) => totalPrice + price, 0);
  return `Total: €${amount}`;
}
```
```

The method Object.values returns an array with all the property values from an object:

```
console.log(Object.values(object)) // [ 1.75, 0.99, 0.79, 0.98, 2.99 ]
```

In this case, it is very useful because we need only the values. As Object.values returns an array, we can call the .reduce method to calculate the sum of all prices.